pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+# Auxiliary Files
+PHEPs may include auxiliary files such as diagrams. Such files should be named `phep-XXXX-Y.ext`, where "XXXX" is the PHEP number, "Y" is a serial number (starting at 1), and "ext" is replaced by the actual file extension (e.g. "png").
+Alternatively, all support files may be placed in a subdirectory called `phep-XXXX`, where "XXXX" is the PHEP number. When using a subdirectory, there are no constraints on the names used in files.
+
+# Changing Existing PHEPs
+Draft PHEPs are freely open for discussion and proposed modification, at the discretion of the authors, until submitted for review and resolution. Substantive content changes should generally be proposed on the PHEP's discussion thread listed in its `Discussions-To` header. For guidance on modifying other PHEPs, consult the [PHEP Maintenance](#phep-maintenance) section.
+
+When in doubt, please check first with the PHEP author and/or a PHEP editor.
+
+# Transferring PHEP Ownership
+<a name="transferring-phep-ownership"></a>
+It occasionally becomes necessary to transfer ownership of PHEPs to a new champion. In general, it is preferable to retain the original author as a co-author of the transferred PHEP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the PHEP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad reason to transfer ownership is because the author doesn't agree with the direction of the PHEP. One aim of the PHEP process is to try to build consensus around a PHEP, but if that's not possible, an author can always submit a competing PHEP.
+
+If you are interested in assuming ownership of a PHEP, you can also do this via the pull request discussion thread. Mention both the original author and `@heliophysicsPy/phep-editors` in a comment on the pull request. (If the original author's GitHub username is unknown, use email.) If the original author doesn't respond in a timely manner, the PHEP editors will make a unilateral decision, which can be reversed.
+
+# PHEP Editor Responsibilities & Workflow
+<a name="phep-editor-responsibilities-workflow"></a>
+A PHEP editor must be added to the `@heliophysicsPy/pep-editors` group on GitHub and must watch the `standards` repository.
+
+Other community members with write access to the `standards` repository may handle the tasks that would normally be taken care of by the PHEP editors. Alternately, even those with write access may request assistance from PHEP editors by mentioning `@heliophysicsPy/phep-editors` on GitHub.
+
+For each new PHEP that comes in an editor does the following:
```

Comment:
I'd add a comma: `For each new PHEP that comes in, an editor does the following:`